addOns/replacer/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+ - Intercept and change non-ASCII payload (Issue 5328)
```

Comment:
IMO we should not use "non-ASCII", it's misleading they are in the ASCII charset (e.g. control chars).
How about?
```markdown
### Added
 - Allow byte replacement using hexadecimal escapes (Issue 5328).
```